x/fundraising/keeper/auction.go

```diff
@@ -303,7 +304,8 @@ func (k Keeper) UpdateAllowedBidder(ctx sdk.Context, auctionId uint64, bidder sd
 	return nil
 }
 
-// AllocateSellingCoin releases designated selling coin from the selling reserve account.
+// AllocateSellingCoin allocates allocated selling coin for all matched bids in MatchingInfo and
+// releases them from the selling reserve account.
 func (k Keeper) AllocateSellingCoin(ctx sdk.Context, auction types.AuctionI, mInfo MatchingInfo) error {
```

Comment:
How about using `ReleaseSellingCoin` instead of `AllocateSellingCoin` in order to keep consistency with below?

Reply:
As discussed offline, we decide to use the following function names:
- AllocateSellingCoin
- ReleaseVestingSellingCoin
- RefundRemainingSellingCoin
- RefundRemainingPayingCoin